integration_tests/features/steps/basic_flow.py

```diff
@@ -0,0 +1,107 @@
+import time
+
+from behave import given, then, when
+from web3 import HTTPProvider, Web3
+
+from plasma_cash.dependency_config import container
+
+client = container.get_client()
+root_chain = container.get_root_chain()
+web3 = Web3(HTTPProvider('http://localhost:8545'))
+
+userA = '0xb83e232458a092696be9717045d9a605fb0fec2b'
+userB = '0x08d92dca9038ea9433254996a2d4f08d43be8227'
+userA_key = '0xe4807cf08191b310fe1821e6e5397727ee6bc694e92e25115eca40114e3a4e6b'
+eth_currency = '0x0000000000000000000000000000000000000000'
+submit_block_sig = '0xa18969817c2cefadf52b93eb20f917dce760ce13b2ac9025e0361ad1e7a1d448'
+uid = 1693390459388381052156419331572168595237271043726428428352746834777341368960
+
+DEPOSIT_TX_BLOCK = 1  # only one block is generated after deposit
+TRANSFER_TX_BLOCK = 2
+
+
+def wei_to_eth(amount_in_wei):
+    return amount_in_wei / 10 ** 18
+
+
+def address_equals(address1, address2):
+    if address1[:2] == '0x':
+        address1 = address1[2:]
+    if address2[:2] == '0x':
+        address2 = address2[2:]
+
+    # address is case insensitive
+    return address1.lower() == address2.lower()
```

Comment:
Minor. But comparing the address in checksum address might be better?
https://web3py.readthedocs.io/en/v3.16.5/overview.html#Web3.toChecksumAddress